README.md

```diff
@@ -15,12 +15,12 @@ See below for an example of how to install Node.js.
 
 - `version` {string} {optional} The version of the CLI to install. Defaults to
   the latest version if not specified.
-- `workspace-id` {string} {optional} A workspace id to configure.  If provided, all future calls to the CLI will use this workspace by default.
```

Comment:
On second thought, it looks like more of an issue with the impl than the docs. The `github-run-mabl-tests` Action uses `*-id`, so it would be consistent for this mabl Action to do the same. In the past we've simply made the impl accept both input variants for changes like this, to keep things backwards compatible. There is a way to log a warning, so users could know to change to the correct param.

Reply:
There was a code change to allow `workspace`, `workspace-id` and `workspace_id` to work correctly [here](https://github.com/mablhq/setup-mabl-cli/blob/634d46ad5598b2630fd0001b542d7cf25311e428/src/index.ts).

Given this, @hgsgtk, I think the edit needed in this PR is actually to `actions.yml` (where GitHub is validating the input flag names), to add `workspace-id` and mark `workspace` as deprecated. Thanks for your contribution!